pixeldata/src/gdcm.rs

```diff
@@ -0,0 +1,257 @@
+    // "pydicom/OT-PAL-8-face.dcm",
+    // "pydicom/SC_rgb_16bit_2frame.dcm",
+    // "pydicom/SC_rgb_2frame.dcm",
+    // "pydicom/SC_rgb_32bit.dcm",
+    // "pydicom/SC_rgb_32bit_2frame.dcm",
+    // "pydicom/SC_rgb_dcmtk_+eb+cy+n1.dcm",
+    // "pydicom/SC_rgb_dcmtk_+eb+cy+n2.dcm",
+    // "pydicom/SC_rgb_dcmtk_+eb+cy+np.dcm",
+    // "pydicom/SC_rgb_dcmtk_+eb+cy+s2.dcm",
+    // "pydicom/SC_rgb_dcmtk_+eb+cy+s4.dcm",
+    // "pydicom/SC_rgb_dcmtk_ebcr_dcmd.dcm",
+    // "pydicom/SC_rgb_dcmtk_ebcyn1_dcmd.dcm",
+    // "pydicom/SC_rgb_dcmtk_ebcyn2_dcmd.dcm",
+    // "pydicom/SC_rgb_dcmtk_ebcynp_dcmd.dcm",
+    // "pydicom/SC_rgb_dcmtk_ebcys2_dcmd.dcm",
+    // "pydicom/SC_rgb_dcmtk_ebcys4_dcmd.dcm",
+    // "pydicom/SC_rgb_expb_16bit_2frame.dcm",
+    // "pydicom/SC_rgb_expb_2frame.dcm",
+    // "pydicom/SC_rgb_expb_32bit.dcm",
+    // "pydicom/SC_rgb_expb_32bit_2frame.dcm",
+    // "pydicom/SC_rgb_rle_16bit_2frame.dcm",
+    // "pydicom/SC_rgb_rle_2frame.dcm",
+    // "pydicom/SC_rgb_rle_32bit.dcm",
+    // "pydicom/SC_rgb_rle_32bit_2frame.dcm",
+    // "pydicom/SC_rgb_small_odd.dcm",
+    // "pydicom/SC_rgb_small_odd_jpeg.dcm",
+    // "pydicom/SC_rgb_jpeg_dcmtk.dcm",
+    // "pydicom/SC_ybr_full_422_uncompressed.dcm",
+    // "pydicom/US1_J2KI.dcm",
+    // "pydicom/US1_J2KR.dcm",
+    // "pydicom/US1_UNCI.dcm",
+    // "pydicom/US1_UNCR.dcm",
+    // "pydicom/badVR.dcm",
+    // "pydicom/bad_sequence.dcm",
+    // "pydicom/color3d_jpeg_baseline.dcm",
+    // "pydicom/eCT_Supplemental.dcm",
+    // "pydicom/empty_charset_LEI.dcm",
+    // "pydicom/emri_small.dcm",
+    // "pydicom/emri_small_RLE.dcm",
+    // "pydicom/emri_small_big_endian.dcm",
+    // "pydicom/emri_small_jpeg_2k_lossless.dcm",
+    // "pydicom/emri_small_jpeg_2k_lossless_too_short.dcm",
+    // "pydicom/emri_small_jpeg_ls_lossless.dcm",
+    // "pydicom/gdcm-US-ALOKA-16.dcm",
+    // "pydicom/gdcm-US-ALOKA-16_big.dcm",
+    // "pydicom/image_dfl.dcm",
+    // "pydicom/liver.dcm",
+    // "pydicom/liver_1frame.dcm",
+    // "pydicom/liver_expb.dcm",
+    // "pydicom/liver_expb_1frame.dcm",
+    // "pydicom/meta_missing_tsyntax.dcm",
+    // "pydicom/mlut_18.dcm",
+    // "pydicom/nested_priv_SQ.dcm",
+    // "pydicom/no_meta.dcm",
+    // "pydicom/no_meta_group_length.dcm",
+    // "pydicom/priv_SQ.dcm",
+    // "pydicom/reportsi.dcm",
+    // "pydicom/reportsi_with_empty_number_tags.dcm",
+    // "pydicom/rtdose.dcm",
+    // "pydicom/rtdose_1frame.dcm",
+    // "pydicom/rtdose_expb.dcm",
+    // "pydicom/rtdose_expb_1frame.dcm",
+    // "pydicom/rtdose_rle.dcm",
+    // "pydicom/rtdose_rle_1frame.dcm",
+    // "pydicom/rtplan.dcm",
+    // "pydicom/rtplan_truncated.dcm",
+    // "pydicom/rtstruct.dcm",
+    // "pydicom/test-SR.dcm",
+    // "pydicom/vlut_04.dcm",
+])]
+    fn test_parse_dicom_pixel_data(value: &str) {
+        let test_file = dicom_test_files::path(value).unwrap();
+        println!("Parsing pixel data for {:?}", test_file);
```

Comment:
This should work, if you wish to keep it.
```suggestion
        println!("Parsing pixel data for {}", test_file.display());
```